### PR TITLE
Add exclude_fields option

### PR DESCRIPTION
**Pull request by TomasVotruba**
I finally found some time :)

Ref original issue at EasyAdminBundle: https://github.com/EasyCorp/EasyAdminBundle/issues/2325

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -228,12 +228,45 @@ Easy configurable with custom list actions by adding a `confirm` key :
...
+    public $name;
+    
+    public $references;
+}
+```
+
+It will show all fields but those mentioned in `exclude_fields`:
+
+```yaml
+easyadmin:
```
[the remark this answers is not in the document]
Comment (TomasVotruba) [reply README.md]: It explains what it does in example. It's must have for clear and fast understanding.

File: src/Configuration/ExcludeFieldsConfigPass.php [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+     * @return mixed[]
+     */
+    public function process(array $backendConfig): array
+    {
+        if (!isset($backendConfig['entities'])) {
+            return $backendConfig;
+        }
+
+        foreach ($backendConfig['entities'] as $entityName => $entityConfig) {
+            if (!isset($entityConfig['form']['exclude_fields'])) {
```
[the remark this answers is not in the document]
Comment (TomasVotruba) [reply src/Configuration/ExcludeFieldsConfigPass.php]: I use it on `form` only.

If you need to extends in some way, please do so. I don't have much the experience of using EasyAdminBundle in this way.

[the remark this answers is not in the document]
Comment (alterphp) [reply src/Configuration/ExcludeFieldsConfigPass.php]: it's exactly the same (new is only for creation route, edit for edition and form for both). I can make the changes if you want (I planned to make the PR anyway ;-) )

**Comment by alterphp** [comment]
Hi @TomasVotruba, I added the requested changes in #58

**Comment by TomasVotruba** [comment]
Hey, thanks for taking care of this :+1: 

Btw, do you write some posts about these? People should know :)

**Comment by alterphp** [comment]
I have no time for writing blog posts (I did in the past). And I hope some features will go straight to the EasyAdminBundle itself, I think some of them are worth it.
Maybe some developer will do, or at least it will continue to naturally gain reputation ;-)

**Comment by TomasVotruba** [comment]
I see. I already wrote about the `exclude_fields` feature itself (in rather how-to-solve-unknown stuff mood).

I'll use the package for a while to get to know it better.

**Pull request closed without merging**